src/assets/usage-examples-code/sprites/create_sprite/create_sprite-1-create-sprite.cs

```diff
@@ -0,0 +1,38 @@
+/*
+    usage example: creating a sprite
+*/
+
+using System;
+using SplashKitSDK;
+
+public class Program
```

Comment:
Update the C# program code to use top level statements only.

Reply:
HI @breezy-codes I have commited the changes suggested by you.